### DISCUSSION TITLE
Timeout with pull function

### DISCUSSION THREAD

**Discussion by GeiOoo** [czproject/git-php#98] [Q&A]
Hey, i want to update the currently checked out branch by calling ``->pull('origin')``. This always results in a timeout without any exceptions. 

I am able to change branches with ``->checkout(...)``, so i think my setup is correct at this point.
Am i missing something?

**Comment by janpecha** [comment]
Hello, I think it can be trouble with authentication (Git waits for user/password input). Try something from Troubleshooting section in readme https://github.com/czproject/git-php?tab=readme-ov-file#remotes
